e2e/specs/add-appointments.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { expect } from '@playwright/test';
+import { generateRandomPatient, deletePatient, type Patient } from '../commands';
+import { test } from '../core';
+import { AppointmentsPage } from '../pages';
+
+let patient: Patient;
+
+test.beforeEach(async ({ api }) => {
+  patient = await generateRandomPatient(api);
+});
+
+test('Add appointment for a patient, edit the added appointment and cancel it', async ({ page, api }) => {
+  const appointmentsPage = new AppointmentsPage(page);
+  await test.step('When I click on appointment tab', async () => {
+    await appointmentsPage.goto(patient.uuid);
+  });
+
+  await test.step('And I click on the “Add” button', async () => {
+    await page.click('button:has-text("Add")');
+  });
+
+  //having trouble with how to test the Schedule appointment workspace
```

Comment:
I tried to do [this](https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-have-text), but raising an error in runtime.
```
 await test.step('Then I should see the Schedule appointment workspace', async () => {
    await expect(page).toHaveText('Schedule appointment');
  });
```

Reply:
Can you share the error message?

Reply:
Please refer to this spec https://github.com/openmrs/openmrs-esm-patient-chart/blob/main/e2e/specs/start-visit.spec.ts